contracts/Core/Parameters.sol

```diff
@@ -37,39 +38,21 @@ contract Parameters is ACL, Constants, IParameters {
         penaltyNotRevealNum = _penaltyNotRevealNumerator;
     }
 
-    function setPenaltyNotRevealDeom(uint16 _penaltyNotRevealDenom) external onlyRole(DEFAULT_ADMIN_ROLE) {
-        emit ParameterChanged(msg.sender, "penaltyNotRevealDenom", penaltyNotRevealDenom, _penaltyNotRevealDenom, block.timestamp);
-        penaltyNotRevealDenom = _penaltyNotRevealDenom;
-    }
-
-    function setSlashPenaltyNum(uint16 _slashPenaltyNumerator) external onlyRole(DEFAULT_ADMIN_ROLE) {
-        emit ParameterChanged(msg.sender, "slashPenaltyNum", slashPenaltyNum, _slashPenaltyNumerator, block.timestamp);
-        slashPenaltyNum = _slashPenaltyNumerator;
-    }
-
-    function setSlashPenaltyDenom(uint16 _slashPenaltyDenominator) external onlyRole(DEFAULT_ADMIN_ROLE) {
-        emit ParameterChanged(msg.sender, "slashPenaltyDenom", slashPenaltyDenom, _slashPenaltyDenominator, block.timestamp);
-        slashPenaltyDenom = _slashPenaltyDenominator;
-    }
-
-    function setBurnSlashNum(uint16 _burnSlashNum) external onlyRole(DEFAULT_ADMIN_ROLE) {
-        emit ParameterChanged(msg.sender, "burnSlashNum", burnSlashNum, _burnSlashNum, block.timestamp);
-        burnSlashNum = _burnSlashNum;
+    function setSlashParams(
+        uint16 _bounty,
+        uint16 _burn,
+        uint16 _keep
+    ) external onlyRole(DEFAULT_ADMIN_ROLE) {
+        require(_bounty + _burn + _keep <= 10000, "Slash nums addtion exceeds 10000");
```

Comment:
10000 should be replaced by baseDenominator

Reply:
sure

Reply:
done https://github.com/razor-network/contracts/pull/471/commits/f7f32af3fa82f9dafca4e3750054cf79453d2d8b